skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/PdlService.kt

```diff
@@ -119,6 +135,9 @@ class PdlService(config: Config, authService: AzureADService) : ServiceStatus {
                     variables = FnrVariables(fnr)
                 )
             )
+            headers {
+                set("Behandlingsnummer", call.attributes[AttributeKey("saktype")]) // Settes av AuthorizeAnsattSakTilgang
```

Comment:
Et problem med denne løsningen er at vi kan nå risikere at "Behandlingsnummer"-header blir satt til null, spesielt siden SakType-enum har flere lovlige verdier enn hva som har mapping til Behandlingsnummer.

Reply:
Ja enig det er bare tidsspørsmål før det eksploderer, men på den andre siden , burde vi hatt behandlingsnummer på resterende eller tenker du å defaulte til alderspensjon eller noe sånnt på de vi ikke har ?

Reply:
Jeg tenker vi burde feile med en feilmelding om at vi ikke har behandlingsgrunnlag for saker med saktyper vi ikke har behandlingsnummer for.